src/renderer/src/lib/modal/ModalInstallForm.svelte

```diff
@@ -41,6 +45,10 @@
 		}
 	};
 
+	$: duplicateAppId = $installedApps.data?.some((extendedAppInfo) => extendedAppInfo.appInfo.installed_app_id === formData.appId);
```

Comment:
Minor refactor to make the code concise

```suggestion
	$: duplicateAppId = $installedApps.data?.some(({appInfo}) => appInfo.installed_app_id === formData.appId);
```

Reply:
agreed, thanks!